src/app/ui/errordialog.cpp

```diff
@@ -0,0 +1,25 @@
+#include "errordialog.h"
+#include "ui_errordialog.h"
+
+ErrorDialog::ErrorDialog(QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::ErrorDialog)
+{
+    ui->setupUi(this);    
+    this->setFixedSize(400,300);
+}
+
+ErrorDialog::~ErrorDialog()
+{
+    delete ui;
+    /*!
+     * \brief connect
+     * Connect performs slot on_pushButtonOk_clicked() after pushing on button Ok
+     */
+    connect(pushButtonOk,SIGNAL(clicked(bool)),this,SLOT(on_pushButtonOk_clicked()));
```

Comment:
Your slot is named like you using auto connection feature. But you are explicitly make connection. Why?